internal/commands/push/inputs.go

```diff
@@ -47,15 +49,25 @@ func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 		searchPath = profile.WorkingDirectory
 	}
 
+	searchPathAbs, err := filepath.Abs(searchPath)
+	if err != nil {
+		return err
+	}
+	_, err = os.Stat(searchPathAbs)
+	if os.IsNotExist(err) {
+		return errProjectInvalid{path: searchPath}
+	}
+
 	app, err := local.LoadAppConfig(searchPath)
 	if err != nil {
 		return err
 	}
 
+	if app.RootDir == "" {
+		return errProjectInvalid{path: searchPath, pathExists: true}
```

Comment:
[style/go]  no change needed here, just figured i'd take this opportunity to point out some go things:

in case you weren't aware, this error could also be created via
```go
return errProjectInvalid{searchPath, true}
```

basically if/when you are instantiating a struct with all of its fields populated, you can specify the values without the respective `fieldName: ` in front of it.

the only reason for it would be making things potentially read easier.  however, there is a drawback of it to be aware of: there's a chance this syntax can inadvertently introduce a bug.  if a field is added or removed, the go compiler will catch that here (b/c it has the incorrect number of fields).  if the fields are re-arranged, the go compiler would _also_ catch that for this instance b/c the fields are different types.  but if you have all `string` (or same typed) fields, then using this syntax can be dangerous as you'll silently change which fields you are assigning the values towards

anyways, ignore me if you knew all of this already.  just thought i'd take a second to highlight some neat "go things" and the gotchas associated with them